webapp_v2/src/components/singlestudy/Commands/Edition/DraggableCommands/CommandListView.tsx

```diff
@@ -0,0 +1,155 @@
+import React, { useEffect, useRef } from "react";
+import { FixedSizeList, areEqual, ListChildComponentProps } from "react-window";
+import {
+  DragDropContext,
+  Droppable,
+  Draggable,
+  OnDragEndResponder,
+} from "react-beautiful-dnd";
+import { styled } from "@mui/material";
+import { CommandItem } from "../CommandTypes";
+import CommandListItem from "./CommandListItem";
+import { scrollbarStyle } from "../../../../../theme";
+
+const StyledList = styled(FixedSizeList)(({ theme }) => ({
+  ...scrollbarStyle,
+  "&::-webkit-scrollbar-thumb": {
+    backgroundColor: theme.palette.secondary.main,
+    outline: "1px solid slategrey",
```

Comment:
remove outline or set it to the same color as background

Reply:
also there is already the "&::-webkit-scrollbar-thumb" property in scrollbarStyle